socket.c

```diff
@@ -932,6 +933,7 @@ static int cache_and_validate_cert(nasp_socket *sc, char *key)
 	// Cert found in the cache use that
 	else
 	{
+		pr_err("nasp: found cert for id[%s]", key);
```

Comment:
```suggestion
		pr_info("nasp: found cert for id[%s]", key);
```

Reply:
this gonna be fix in the logging related PR